### PR TITLE
DriveWithXbox

### PR DESCRIPTION
**Pull request by EliyaFishman**

### CODE REVIEW

**Comment by EliyaFishman** [comment]
[Drivetrain Command: xbox drive](https://trello.com/c/Nd6BZllt/10-drivetrain-command-xbox-drive)

File: src/main/java/frc/robot/subsystems/drivetrain/DriveWithXbox.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package frc.robot.subsystems.drivetrain;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import java.util.function.DoubleSupplier;
+
+import static frc.robot.Robot.drivetrain;
+
+
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/subsystems/drivetrain/DriveWithXbox.java]: blank line

File: src/main/java/frc/robot/subsystems/drivetrain/DriveWithXbox.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+public class DriveWithXbox extends CommandBase {
+
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/subsystems/drivetrain/DriveWithXbox.java:11]: blank line

File: src/main/java/frc/robot/subsystems/drivetrain/DriveWithXbox.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        this.yInput = yInput;
+    }
+
+
+    @Override
+    public void execute() {
+        drivetrain.trigonCurvatureDrive(xInput.getAsDouble(), yInput.getAsDouble());
+    }
+
+
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/subsystems/drivetrain/DriveWithXbox.java]: blank line

File: src/main/java/frc/robot/subsystems/drivetrain/Drivetrain.java [hunk 4]
```diff
@@ -62,6 +63,7 @@ public Drivetrain() {
 
         kinematics = new DifferentialDriveKinematics(robotConstants.drivetrainConstants.kWheelBaseWidth);
         odometry = new DifferentialDriveOdometry(Rotation2d.fromDegrees(getAngle()));
+        SmartDashboard.putData("Drivetrain/DifferentialDrive", drivetrain);
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/subsystems/drivetrain/Drivetrain.java]: SmartDashboard.putData("Drivetrain/DifferentialDrive", drivetrain);
do as tuning method, and use it if the DriveWithXbox command gets true in constructor for tuning mode

**Review by HilelAmiel — CHANGES_REQUESTED** [review]
some things

**Review by HilelAmiel — APPROVED** [review]
now done

**Comment by EliyaFishman** [comment]
Why aren't you merging this?

**Pull request merged**